test/e2e/multik8s/testutil/util.go

```diff
@@ -99,6 +101,22 @@ func Kubectl(clusterNo int, input []byte, args ...string) ([]byte, []byte, error
 	return execAtLocal(ctx, fields[0], input, fields[1:]...)
 }
 
+func Make(args ...string) error {
```

Comment:
```suggestion
func RunMakeCommand(args ...string) error {
```

`Make` looks too generic name.

Reply:
I fixed this method and made it private.